### ISSUE TITLE
500px fails to fetch photos

### ISSUE THREAD

**Issue by mvysny** [mvysny/photocloud-frame-slideshow#85]
```
java.net.MalformedURLException: no protocol: ["https:\/\/drscdn.500px.org\/photo\/249396007\/m%3D1600_k%3D1_a%3D1\/v2?client_application_id=21143&webp=true&sig=d03a3ea844b8507f454407d19abadb3c96bab4fbb3a4906dc64d7ca67fb4777b"]
                               at java.net.URL.<init>(URL.java:589)
                               at java.net.URL.<init>(URL.java:486)
```

**Comment by mvysny** [comment]
What the... did just 500px started to escape `/` ?

**Comment by mvysny** [comment]
Nope, it's just 500px now provides a JSONArray of URLs instead of just a plan URL, contrary on what they promise at https://github.com/500px/legacy-api-documentation/blob/master/endpoints/photo/GET_photos.md .
Fixed in PhotoCloud 1.11.2
